src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -136,6 +144,12 @@ public AddressBook load() throws StorageOperationException {
         // create empty file if not found
         } catch (FileNotFoundException fnfe) {
             final AddressBook empty = new AddressBook();
+            // XXX hack to create an empty file since we now do not allow non existent files
+            try {
+                Files.createFile(path);
+            } catch (IOException e) {
+                throw new StorageOperationException("Cannot create file!");
+            }
```

Comment:
Based on what I have learned from
https://github.com/nus-cs2103-AY1617S2/addressbook-level2/pull/431#discussion_r99479917

This means that the program no longer stops if the program starts with no file at the start, but perhaps I might still want to create an empty file before attempting to save so I can handle the program **start** file missing situation vs program **executing** missing situation separately